RxMediaPicker/RxMediaPicker.swift

```diff
@@ -92,79 +93,78 @@ public enum RxMediaPickerError: Error {
                 picker.cameraFlashMode = flashMode
             }
             
-            self.delegate?.present(picker: picker)
+            self.presentPicker(picker)
             
             return Disposables.create()
-        })
+        }
     }
     
     open func selectImage(source: UIImagePickerControllerSourceType = .photoLibrary, editable: Bool = false) -> Observable<(UIImage, UIImage?)> {
         
-        return Observable.create { observer in
+        return Observable.create { [unowned self] observer in
             self.currentAction = RxMediaPickerAction.photo(observer: observer)
             
             let picker = UIImagePickerController()
             picker.sourceType = source
             picker.allowsEditing = editable
             picker.delegate = self
             
-            self.delegate?.present(picker: picker)
+            self.presentPicker(picker)
             
             return Disposables.create()
         }
     }
     
     func processPhoto(info: [String : AnyObject], observer: AnyObserver<(UIImage, UIImage?)>) {
-        if let image = info[UIImagePickerControllerOriginalImage] as? UIImage {
-            let editedImage: UIImage? = info[UIImagePickerControllerEditedImage] as? UIImage
-            observer.on(.next(image, editedImage))
-            observer.on(.completed)
-        } else {
+        guard let image = info[UIImagePickerControllerOriginalImage] as? UIImage,
+              let editedImage = info[UIImagePickerControllerEditedImage] as? UIImage else {
             observer.on(.error(RxMediaPickerError.generalError))
```

Comment:
I see, that makes sense. No, I don't have any idea what the other `RxSwiftCommunity` repos do. `raywenderlich`'s style guide says to use [`throw` instead of doing a return](https://github.com/raywenderlich/swift-style-guide#failing-guards) if theres going to be an error.

Can you throw inside a function that has an observer and have RxSwift recognize that you're trying to onError the observer? Or does it need to be explicit? (I know they recognize `throw`'s in operators like `map`